res/values-sq/strings.xml

```diff
@@ -662,59 +660,59 @@
+    <string name="nebulas_">"Nebulas."</string>
     <string name="Incompatible_versions_">"Versione të papajtueshme."</string>
     <string name="Update_Available_">"Update Available!"</string>
-    <string name="VIEW_INVITES">"fton VIEW"</string>
-    <string name="CLAN_INVITES">"fton CLAN"</string>
+    <string name="VIEW_INVITES">"FTON PAMJE"</string>
+    <string name="CLAN_INVITES">"FTON KLAN"</string>
     <string name="Cancel_Invitation">"Anulo Ftesë"</string>
     <string name="DOWNLOAD">"SHKARKO"</string>
     <string name="REJECT_ALL_FRIEND_INVITATIONS_">"POSHTË gjitha mik ftesa?"</string>
-    <string name="suns_">"suns."</string>
-    <string name="Sun_Collector">"Sun Collector"</string>
-    <string name="Sun_Hoarder">"Sun grumbullues"</string>
-    <string name="Sun_Mogul">"Sun Mogul"</string>
-    <string name="Collect_1_000_suns_">"Mblidhni 1.000 suns."</string>
-    <string name="Collect_5_000_suns_">"Mblidhni 5000 suns."</string>
-    <string name="Collect_10_000_suns_">"Mblidhni 10.000 suns."</string>
+    <string name="suns_">"Diellë."</string>
+    <string name="Sun_Collector">"Dielli Koleksionist"</string>
+    <string name="Sun_Hoarder">"Dielli Grumbullues"</string>
+    <string name="Sun_Mogul">"Dielli Manjati"</string>
+    <string name="Collect_1_000_suns_">"Mblidhni 1.000 Diella."</string>
+    <string name="Collect_5_000_suns_">"Mblidhni 5.000 Diella."</string>
+    <string name="Collect_10_000_suns_">"Mblidhni 10.000 Diella."</string>
     <string name="Reward_Videos_">"Shpërblim Videos!"</string>
     <string name="Free_Plasma_">"pa pagesë Plazma!"</string>
-    <string name="moons_">"hënave."</string>
-    <string name="Moon_Collector">"hënë Collector"</string>
-    <string name="Moon_Hoarder">"hëna grumbullues"</string>
-    <string name="Moon_Mogul">"hëna Mogul"</string>
-    <string name="Collect_1_000_moons_">"Mblidhni 1.000 hënave."</string>
-    <string name="Collect_5_000_moons_">"Mblidhni 5000 hënave."</string>
-    <string name="Collect_10_000_moons_">"Mblidhni 10.000 hënave."</string>
+    <string name="moons_">"Hënë."</string>
+    <string name="Moon_Collector">"Hëna Koleksionist"</string>
+    <string name="Moon_Hoarder">"Hëna Grumbullues"</string>
+    <string name="Moon_Mogul">"Hëna Manjati"</string>
+    <string name="Collect_1_000_moons_">"Mblidhni 1.000 Hënave."</string>
+    <string name="Collect_5_000_moons_">"Mblidhni 5.000 Hënave."</string>
+    <string name="Collect_10_000_moons_">"Mblidhni 10.000 Hënave."</string>
     <string name="BLUETOOTH">"BLUETOOTH"</string>
     <string name="HOST">"MIKPRITËS"</string>
     <string name="This_device_does_not_support_bluetooth_">"Kjo pajisje nuk mbështet Bluetooth."</string>
     <string name="member_permissions">"Permissions anëtare"</string>
-    <string name="edit">"EDIT"</string>
-    <string name="start_clan_war">"Filloni Clan War"</string>
-    <string name="join_clan_war">"Join Clan War"</string>
+    <string name="edit">"edit"</string>
```

Comment:
If they're kept as capital, please don't remove them. You don't need to base off on blue colour text on every line. For help, compare `res/values` (English file) to avoid this kind of inconsistency.